docs/UserGuide.md

```diff
@@ -131,52 +131,58 @@ Simply click on any command below to learn more about it!
 | [`edit INDEX ...`](#edit) | Edit a contact |
 | [`edit INDEX [t/TAG]`](#edit_tag) | Create / Edit tags for existing contacts |
 | [`edit INDEX t/`](#delete_tag) | Delete all tags from a contact |
-| [`edit INDEX [p/PLATFORM]`](#edit_platform) | Modify social media handles for a contact |
-| [`edit INDEX [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​`](#edit_dates) | Modify dates for a contact |
+| [`edit INDEX [p/PLATFORM]`](#edit_platform) | Modify social media handles of a contact |
+| [`edit INDEX [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​`](#edit_dates) | Modify associated dates of a contact |
 | [`delete INDEX`](#delete) | Delete a contact |
 | [`clear`](#clear) | Delete all contacts |
 | [`find NAME`](#find) | Find a contact by name |
 | [`find t/TAG`](#find_tag) | Find a contact through associated tags |
 | [`find p/PLATFORM`](#find_platform) | Find a contact through associated social media platforms |
 | [`picture INDEX`](#picture) | Modify profile picture of a contact |
 | [`share INDEX`](#share) | Copy contact information to system clipboard |
-| [`pin INDEX`](#pin) | Pin contact card on the top of the person list |
-| [`unpin INDEX`](#unpin) | Unpin contact card from the top of the person list |
+| [`pin INDEX`](#pin) | Pin contact card on the top of the contact list |
+| [`unpin INDEX`](#unpin) | Unpin contact card from the top of the contact list |
 | [`remark INDEX r/[TEXT]`](#remark) | Add special notes about a contact |
 | [`help`](#help) | Access User Guide |
 | [`help COMMAND`](#help_command) | Access in-app guidance for selected commands |
 | [`exit`](#exit) | Exit the program |
 
 
-### Add a person: `add n/NAME p/PHONE_NUMBER ...` <a name="add"></a>
+### Add a contact: `add n/NAME p/PHONE_NUMBER ...` <a name="add"></a>
 
-Adds a person to SociaLite.
+Adds a contact to SociaLite.
 
 <div markdown="block" class="alert alert-primary">
 **:mag_right: Format:**
 `add n/NAME p/PHONE_NUMBER [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
-* A person can have any number of tags and dates (including 0).
-* To add a contact to SociaLite, you need to provide the **name** and a **unique phone number** for the person!
+* To add a contact to SociaLite, you must provide the **name of the person** and a **unique phone number**!
+* A contact can have any number of tags and dates (including 0).
+* A contact can have a handle under each of these 5 social media platforms: *Facebook, Instagram, Telegram, Tiktok & Twitter*.
+
 </div>
 
 <div markdown="block" class="alert alert-success">
-**:heavy_check_mark: Example:** `add n/Annie Wan p/98571082 t/Friend fb/annie.wan ig/ann.wan tele/annie tiktok/annie.wan2 date/birthday:1999-02-12:yearly`
+**:heavy_check_mark: Example:** `add n/David Tan p/91082942`
```

Comment:
It's ok to reorder the sequence of commands but it is advisable to reuse the entire example (refer to line 170 in the current version) i.e.: `add n/David Tan p/91082942 t/neighbour fb/tan.david tele/d_av_id` instead of the truncated one you have modified here. This is because the `find David` example under [**Locate contacts by name: `find NAME`**](https://stanley-1.github.io/tp/UserGuide.html#find) has a screenshot which displays all these details for David Tan.